Test suite broken by the new year

Some tests depend on the precise current year and have been broken since yesterday:

```
=================================== FAILURES ===================================
_____________ test_ufo_fontinfo_parsing[empty-key-name-fdarray--0] _____________

file = 'empty-key-name-fdarray', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmpzz7u_rl6', '/build/tmp5foz5qjm'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmpzz7u_rl6
+++ /build/tmp5foz5qjm
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
________________ test_ufo_fontinfo_parsing[missing-key-name--0] ________________

file = 'missing-key-name', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmpk7hnkgku', '/build/tmphiil_gry'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmpk7hnkgku
+++ /build/tmphiil_gry
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
_______________ test_ufo_fontinfo_parsing[missing-key-name-2--0] _______________

file = 'missing-key-name-2', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmp973x0pr8', '/build/tmpmcvjwdz6'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmp973x0pr8
+++ /build/tmpmcvjwdz6
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
______________ test_ufo_fontinfo_parsing[missing-key-value-2--0] _______________

file = 'missing-key-value-2', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmpd6_xhb9v', '/build/tmpewjtpsn3'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmpd6_xhb9v
+++ /build/tmpewjtpsn3
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
_______________ test_ufo_fontinfo_parsing[bluesarray-string--0] ________________

file = 'bluesarray-string', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmpomsk4dnw', '/build/tmph3rkkc0e'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmpomsk4dnw
+++ /build/tmph3rkkc0e
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
___________ test_ufo_fontinfo_parsing[switched-string-and-array--0] ____________

file = 'switched-string-and-array', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmp_jelw755', '/build/tmpvfeib5fk'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmp_jelw755
+++ /build/tmpvfeib5fk
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
___________________ test_ufo_fontinfo_parsing[empty-dict--0] ___________________

file = 'empty-dict', msg = b'', ret_code = 0

    @pytest.mark.parametrize('file, msg, ret_code', [
        ("empty-key-name", b'', 0),
        ("empty-key-name-fdarray", b'', 0),
        ("invalid-key-name", b'', 0),
        ("missing-key-name", b'', 0),
        ("missing-key-name-2", b'', 0),
        ("empty-key-value", b'Warning: Encountered empty <string> for fontinfo ' +
                            b'key postscriptFontName. Skipping', 0),
        ("missing-key-value", b'', 0),
        ("missing-key-value-2", b'', 0),
        ("bluesarray-string", b'', 0),
        ("fontmatrix-string", b'', 0),
        ("switched-string-and-array", b'', 0),
        ("empty-dict", b'', 0)
    ])
    def test_ufo_fontinfo_parsing(file, msg, ret_code):
        folder = "ufo-fontinfo-parsing/"
        ufo_input_path = get_input_path(folder + file + ".ufo")
        expected_path = get_expected_path(folder + file + ".subset")
        output_path = get_temp_file_path()
        arg = CMD + ['-s', '-e', '-a', '-o', 't1', '-f',
                     ufo_input_path, output_path]
        stderr_path = runner(arg)
        with open(stderr_path, 'rb') as f:
            output = f.read()
        assert (msg) in output
        if (ret_code == 0):
            expected_path = generate_ps_dump(expected_path)
            output_path = generate_ps_dump(output_path)
>           assert differ([expected_path, output_path])
E           AssertionError: assert False
E            +  where False = differ(['/build/tmpsg6k_0kz', '/build/tmp36s4m83l'])

tests/tx_test.py:1249: AssertionError
----------------------------- Captured stdout call -----------------------------
--- /build/tmpsg6k_0kz
+++ /build/tmp36s4m83l
@@ -3,3 +3,3 @@
 %ADOt1write: (1.0.35)
-%%Copyright: Copyright 2022 Adobe System Incorporated. All rights reserved.
+%%Copyright: Copyright 2023 Adobe System Incorporated. All rights reserved.
 %%DocumentNeededResources: ProcSet (CIDInit)
=========================== short test summary info ============================
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[empty-key-name-fdarray--0]
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[missing-key-name--0] - Ass...
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[missing-key-name-2--0] - A...
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[missing-key-value-2--0] - ...
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[bluesarray-string--0] - As...
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[switched-string-and-array--0]
FAILED tests/tx_test.py::test_ufo_fontinfo_parsing[empty-dict--0] - Assertion...
======= 7 failed, 1308 passed, 4 skipped, 8 xfailed in 284.06s (0:04:44) =======
```

--- a/tests/tx_test.py
+++ b/tests/tx_test.py
@@ -1251,7 +1251,7 @@ def test_ufo_fontinfo_parsing(file, msg, ret_code):
     if (ret_code == 0):
         expected_path = generate_ps_dump(expected_path)
         output_path = generate_ps_dump(output_path)
-        assert differ([expected_path, output_path])
+        assert differ([expected_path, output_path, '-s'] + PFA_SKIP)
     else:
         arg = [TOOL, '-t1', '-f', ufo_input_path]
         assert subprocess.call(arg) == ret_code
@@ -1396,7 +1396,7 @@ def test_ufo_contentsplist_parsing(file, msg, ret_code):
     if (ret_code == 0):
         expected_path = generate_ps_dump(expected_path)
         output_path = generate_ps_dump(output_path)
-        assert differ([expected_path, output_path])
+        assert differ([expected_path, output_path, '-s'] + PFA_SKIP)
     else:
         arg = [TOOL, '-t1', '-f', ufo_input_path]
         assert subprocess.call(arg) == ret_code
@@ -1451,7 +1451,7 @@ def test_ufo_glifs_parsing(file, msg, ret_code):
     if (ret_code == 0):
         expected_path = generate_ps_dump(expected_path)
         output_path = generate_ps_dump(output_path)
-        assert differ([expected_path, output_path])
+        assert differ([expected_path, output_path, '-s'] + PFA_SKIP)
     else:
         arg = [TOOL, '-t1', '-f', ufo_input_path]
         assert subprocess.call(arg) == ret_code

